src/main/java/racingCar/RacingCars.java

```diff
@@ -0,0 +1,29 @@
+package racingCar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingCars {
+
+  private final List<RacingCar> racingCarList;
+  int tryCount;
+
+  public RacingCars() {
+    this.racingCarList = new ArrayList<>();
+    this.tryCount = 0;
+  }
+
+  public void add(RacingCar racingCar) {
+    racingCarList.add(racingCar);
+  }
+
+  public void moveAllCars() {
+    racingCarList.forEach(e -> e.tryMove(new Random().nextInt(10)));
+    this.tryCount++;
+  }
+
+  public int[] getAllCarsMoveCount() {
+    return racingCarList.stream().mapToInt(RacingCar::getMoveCount).toArray();
+  }
```

Comment:
```suggestion
  public int[] getAllCarsMoveCount() {
    return racingCarList.stream()
            .mapToInt(RacingCar::getMoveCount)
            .toArray();
  }
```
개행을 통해서 가독성을 높여보는 것은 어떨까요? 

또한 배열은 사이즈가 고정되어 있고 리스트보다 사용하기 어려운데 이번 과정에서 List, Set, Map 과 같은 컬렉션과 친해져 보는 것은 어떨까요?

```java
    return racingCarList.stream()
            .mapToInt(RacingCar::getMoveCount)
            .collect(Collectors.toList());
```